pkg/ingester/metrics.go

```diff
@@ -193,6 +201,18 @@ func newIngesterMetrics(r prometheus.Registerer, createMetricsConflictingWithTSD
 			return 0
 		}),
 
+		inflightQueryRequests: promauto.With(r).NewGaugeFunc(prometheus.GaugeOpts{
+			Name: "cortex_ingester_inflight_query_requests",
```

Comment:
Can we reflect `Max` in the metric itself? Otherwise it is very easy to think it is `Current` inflight requests. Similar to `cortex_ingester_inflight_push_requests`.